packages/runtime/src/types.ts

```diff
@@ -211,6 +219,40 @@ export interface Sign {
   lsig?: LogicSig
 }
 
+export interface ModifyAssetParam extends Sign {
+  type: TransactionType.ModifyAsset
+  fromAccount: AccountSDK
+  assetID: number
+  fields: AssetModFields
+  payFlags: TxParams
+}
+
+export interface FreezeAssetParam extends Sign {
+  type: TransactionType.FreezeAsset
+  fromAccount: AccountSDK
+  assetID: number
+  freezeTarget: AccountAddress
+  freezeState: boolean
+  payFlags: TxParams
+}
+
+export interface RevokeAssetParam extends Sign {
+  type: TransactionType.RevokeAsset
+  fromAccount: AccountSDK
+  recipient: AccountAddress
+  assetID: number
+  revocationTarget: AccountAddress
```

Comment:
Please add a comment in `fromAccount`, `revocationTarget`. `fromAccount` is the clawback address, and the revocation target is the account from which the clawback revokes/retrieves the asset.

Reply:
ok